overlord/snapstate/link_snap_test.go

```diff
@@ -122,6 +122,7 @@ func (s *linkSnapSuite) TestDoUndoLinkSnap(c *C) {
 	t.Set("snap-setup", &snapstate.SnapSetup{
```

Comment:
it's a bit suspicious that this file isn't growing more tests, about do/undo scenarios